src/text_file_functions.cpp

```diff
@@ -435,7 +435,7 @@ QString AOApplication::get_tagged_stylesheet(QString target_tag, QString p_file)
 
 QString AOApplication::get_chat_markdown(QString p_identifier, QString p_chat)
 {
-  QString design_ini_path = get_base_path() + "misc/" + p_chat + "/config.ini";
+  QString design_ini_path = get_base_path() + "misc/" + get_chat(p_chat) + "/config.ini";
```

Comment:
If the behavior changed in #230 is folded into `get_chat` per review, this might lead to unexpected behavior - in fact, one might begin to question what the purpose of the misc folder even is to begin with, if themes can take precedence over misc folders.

Reply:
Well, as far as this fix goes, get_chat_markdown was trying to access a folder named exactly as the character in misc. To me this is an obvious bug and should instead be reading the custom chatbox in misc. To me the order in which get_chat_markdown read ini files is something for another PR, https://github.com/AttorneyOnline/AO2-Client/pull/230 for example.

Reply:
I'll allow this bugfix to pass, but having to manage the precedence orders of individual asset types like this is not setting a good precedent and will take various passes of what they call "un-fucking" later on.

Reply:
I agree, and like I said I can't wait to help you in what I can extinguishing the AO dumpster fire with the 2.9 branch.